example/android/app/google-services.json

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
@shivamsharma2710 Do not commit this file, please. GitHub will raise a security concern

Reply:
Yes, added in the gitignore.